apps/testingaccessibility/src/lib/prisma-api.ts

```diff
@@ -220,15 +223,35 @@ export function getSdk(
       })
 
       if (!merchantCustomer) {
+        Sentry.addBreadcrumb({
+          category: 'commerce',
+          level: Sentry.Severity.Info,
+          message: `adding a merchant customer to the database for user ${user.email}`,
+        })
         merchantCustomer = await ctx.prisma.merchantCustomer.create({
           data: {
-            userId,
+            userId: user.id,
             identifier,
             merchantAccountId,
           },
         })
       }
 
+      const merchantCustomersForUser =
+        await ctx.prisma.merchantCustomer.findMany({
+          where: {
+            user,
+          },
+        })
+
+      if (merchantCustomersForUser.length > 1) {
+        Sentry.addBreadcrumb({
+          category: 'commerce',
+          level: Sentry.Severity.Warning,
+          message: `user [${user.email}] has multiple stripe customers`,
+        })
+      }
```

Comment:
it's currently possible for a given user (email address) to have multiple customers in stripe. this will happen if they make a purchase while logged out in particular